src/components/AddItemForm.js

```diff
@@ -87,24 +96,41 @@ class AddItemForm extends React.Component {
           </label>
           <br />
           <label className="schedule">
-            Schedule:
-            <select
-              className="schedule-btn"
-              value={this.state.next_purchase}
-              onChange={this.handleSchedule}
-            >
-              <option value="7">Soon</option>
-              <option value="14">Kind Of Soon</option>
-              <option value="30">Not Soon</option>
-            </select>
+            How soon do you need to buy {this.state.name}?
           </label>
+          <span className="btn-container">
```

Comment:
I would make this a `div` instead of a span. `span` to me implies it's going to be some kind of inline element. If it's a container for something, I would make it a `div`, barring some other more appropriate tag.